test/unit/model/helpers/installer-test.js

```diff
@@ -157,6 +157,18 @@ describe('Installer', function() {
       });
     });
 
+    it('should resolve as true if no error occurs' , function() {
+      let stub = sandbox.stub(fs, 'createReadStream').yields();
```

Comment:
References for further reading:
* EventEmitter - https://nodejs.org/api/events.html#events_class_eventemitter
* Streams - https://nodejs.org/api/stream.html
* Sinon.js onCall(n) method - http://sinonjs.org/docs/#stub-onCall